src/arrays.js

```diff
@@ -1,61 +1,88 @@
 const getNthElement = (index, array) => {
   // your code here
+  const newIndex = index % array.length;
+  return array[newIndex];
 };
 
 const arrayToCSVString = array => {
   // your code here
+  return array.join(',');
 };
 
 const csvStringToArray = string => {
   // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
   // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
   // your code here
+  const newArray = [...array];
+  newArray.push(element);
+  return newArray;
 };
 
 const removeNthElement = (index, array) => {
   // your code here
+  array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
   // your code here
+  return numbers.map(x => String(x));
```

Comment:
Its a good habit to get into to use more descriptive variable names rather rather something super generic like `x`:
```
  return numbers.map(number => String(number));
```